conf/agent/agent_full.conf

```diff
@@ -310,11 +310,11 @@ plugins {
 #     # bind_address = "localhost"
 
 #     # bind_port: HTTP Port number of the health checks endpoint. Default: 80.
-#     # bind_port = "80"
+#     # bind_port = "8080"
```

Comment:
We should probably leave this as is, since the default is currently 80. We can update the documentation if we ever migrate away from it.

Reply:
OK! I fixed that with https://github.com/spiffe/spire/pull/2015/commits/1fb4a0d24fc665119e7c03a7e45b420ce563914d.